lib/tasks/single_run.rake

```diff
@@ -52,4 +52,19 @@ namespace :single_run do
       application.save
     end
   end
+
+  require 'selection/service/application_distribution'
+  desc '2017-03-25: Distribute Applications to Reviewers'
+  task distribute_applications_to_reviewers: :environment do
```

Comment:
May be personal preference, but as this is not necessarily "just" a *one-off-data-correction-migration*, it could also live in the `selection` task namespace. From how I see it, it can without a problem be run multiple times @beanieboi ?